### DISCUSSION TITLE
task 03: Fill in missing data

### DISCUSSION THREAD

**Discussion by jaayg23** [rramosp/ai4eng.v1#125] [02 - PYTHON]
Buenas noches profesor, quisiera pedirle ayuda con la tare 3 del laboratorio 02.03 ya que no he podido obtener la salida esperada y no logro comprender el por qué
![image](https://user-images.githubusercontent.com/73083558/167747461-50a9cb6b-c279-41ab-9339-49b1f2adbf9f.png)

Y aquí adjunto la salida 
![image](https://user-images.githubusercontent.com/73083558/167747290-d2e06856-bc17-4ed4-b5a7-cfc856f1ca05.png)
Ya que no logro comprender a cabalidad el problema por eso vengo en busca de ayuda. Muchas gracias.

**Comment by Felipe-RA** [comment]
Saludos jaayg23

Estas utilizando la función fillna de pandas cuando se te pide que hagas una implementación propia.

Te sugiero que te implementes una solución propia haciendo uso de `isna()` para esa parte especifica de tu código. 

El objetivo de este task es, en parte, que mejores tu manejo de dataframes.

Ahora, sobre por qué no funciona tu implementación, el problema está en cómo se están rellenando esos na. (Esto lo entenderás mejor cuando hagas tu propio fillna)

Cualquier cosa, por aquí estamos.

Hasta pronto!

**Reply by jaayg23** [reply]
Gracias por la respuesta, pero he implementado el siguiente código con `isna()`  
![image](https://user-images.githubusercontent.com/73083558/167921417-bfe64229-6029-49de-8e05-7a49c77e8f48.png)
Y me aparece aun el mismo error
![image](https://user-images.githubusercontent.com/73083558/167921511-6aa5b52a-09d0-4538-a61c-9e948b84f43b.png)
Gracias de antemano.

**Comment by rramosp** [comment]
tienes que generar una muestra para cada dato ... estas usando la misma
para todos los datos

On Wed, May 11, 2022 at 1:36 PM Jacobo Ayala Giraldo <
***@***.***> wrote:

> Gracias por la respuesta, pero he implementado el siguiente código con
> isna()
> [image: image]
> <https://user-images.githubusercontent.com/73083558/167921417-bfe64229-6029-49de-8e05-7a49c77e8f48.png>
> Y me aparece aun el mismo error
> [image: image]
> <https://user-images.githubusercontent.com/73083558/167921511-6aa5b52a-09d0-4538-a61c-9e948b84f43b.png>
> Gracias de antemano.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/125#discussioncomment-2732551>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIK7TSVKJNJWZPLPNRLVJP4ZFANCNFSM5VTIGNGA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Reply by jaayg23** [reply]
Muchas gracias por la ayuda, gracias a eso lo he solucionado.

**Reply by Felipe-RA** [reply]
Gracias por compartir jaayg23!
